IaC/psmdb.cd/init.groovy.d/cloud.groovy

```diff
@@ -78,6 +78,10 @@ initMap['docker'] = '''
         sudo mount -o noatime ${DEVICE} /mnt
     fi
 
+    cat /etc/hosts > ./hosts
+    echo '10.30.6.9 repo.ci.percona.com' >> ./hosts
+    sudo cp ./hosts /etc/
```

Comment:
sometimes, `/etc/hosts` controlled externaly, so it is bad idea to recreate it
it is simpler to edit it in place to avoid issues
```suggestion
    echo '10.30.6.9 repo.ci.percona.com' | sudo tee -a /etc/hosts
```

Reply:
Tested on rel and reworked 713df27